components/fieldgroups/product/product-brand.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/product-brand",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Product Brand",
+  "type": "object",
+  "auditable": true,
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/product"],
+  "description": "Captures properties related to product branding generic to all industries.",
+  "definitions": {
+    "product-brand": {
+      "properties": {
+        "xdm:productPageURL": {
+          "title": "Product Page URL",
+          "type": "string",
+          "format": "uri",
+          "description": "The URL for the landing page of a product."
+        },
+        "xdm:productImageURL": {
+          "title": "Product Image URL",
+          "type": "string",
+          "format": "uri",
+          "description": "The URL for the primary image of a product."
```

Comment:
To keep the field identical with the other field in `https://ns.adobe.com/xdm/mixins/product-identifiers` field group, change the description to `The URL for the primary product image of the product.`